### PR TITLE
Correct lifecycle comment for A1

### PR DESCRIPTION
**Pull request by msorens**
### :nut_and_bolt: Description: What code changed, and why?

Correct lifecycle comment for A1

### :chains: Related Resources
NA

### :+1: Definition of Done
Corrected info.

### :athletic_shoe: How to Build and Test the Change
NA

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/automate-chef-io/content/docs/migrate.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ To configure Chef Automate 1 backups, see the [Chef Automate 1 Documentation](ht
 Chef Automate 2 includes significant architectural and technical improvements to the core product platform.
 For the initial releases of this new major version, we do not yet support a handful of capabilities that were present in Chef Automate 1.
 If you rely on any of the capabilities listed below, we recommend you continue to use Chef Automate 1.
-Chef continues support of Chef Automate 1 while we develop upgrade paths for these capabilities on the new platform.
+However, note that Chef Automate 1 reached end-of-life on December 31, 2019.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/migrate.md]: can we link to https://docs.chef.io/versions.html

Comment (mjingle) [inline components/automate-chef-io/content/docs/migrate.md]:
```suggestion
Note that Chef Automate 1 reached end-of-life on December 31, 2019.
```

Comment (kagarmoe) [reply components/automate-chef-io/content/docs/migrate.md]: A couple of issues:
1) The EOL needs to be at the top of the paragraph.
2) There is only one Chef Automate in terms of product and marketing; Chef Automate 1 & 2 are useful fictions. We should clarify something along the lines of "The earlier version of Chef Automate, also referred to as Chef Automate 1 and the current version of Chef Automate..." etc.,

Comment (msorens) [reply components/automate-chef-io/content/docs/migrate.md]: @kagarmoe How about you go ahead and update the text? I don't mind at all. 😁

File: components/automate-chef-io/content/docs/migrate.md [hunk 2]
```diff
@@ -109,9 +113,7 @@ To configure Chef Automate 1 backups, see the [Chef Automate 1 Documentation](ht
 
 Chef Automate 2 includes significant architectural and technical improvements to the core product platform.
 For the initial releases of this new major version, we do not yet support a handful of capabilities that were present in Chef Automate 1.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/migrate.md]: i think we should take out this stuff about the initial release, shouldnt we?

Comment (kagarmoe) [inline components/automate-chef-io/content/docs/migrate.md]:
```suggestion
```

**Pull request merged**